components/alert/alert.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alert • PowertrainUI</title>
+  </head>
+  <body>
+    <main class="wrapper">
+      <nav class="wrapper__nav-top">
+        <div class="nav__logo">
+          <a href="#">
+            <h2 class="nav__title">PowertrainUI</h2>
+            <img
+              class="nav__img"
+              src="./images/icon.png"
+              alt="PowertrainUI logo"
+            />
+          </a>
+        </div>
+        <ul class="nav__menu">
+          <li class="nav__menu-item"><a href="#">Get Started</a></li>
+          <li class="nav__menu-item"><a href="#">Docs</a></li>
+        </ul>
+      </nav>
+      <section class="main-content">
+        <aside></aside>
+        <article></article>
+      </section>
+      <footer class="wrapper__footer">
```

Comment:
footer should have it's own space, it shouldn't be inside `main` 

General structure is like 

```html
<body>
  <header></header>
  <main></main>
  <footer></footer>
</body>

```

Reply:
Good catch, will revamp html structure when I work on it!